pkg/clusterresourceoverride/mutator.go

```
@@ -90,13 +91,36 @@ func (m *podMutator) OverrideForceSelinuxRelabel(pod *corev1.Pod) {
+	m.OverrideCPUWithRequest(&container.Resources, container.Name, current)
+}
+
+// Annotates pod with original CPU request value. Annotation provides idempotency for
+// OverrideCPUWithRequest in case of reinvocation. Also preserves original value in case
+// of OverrideCPUWithLimit modifying the request prior to OverrideCPUWithRequest.
+func (m *podMutator) AnnotateOriginalRequest(resources *corev1.ResourceRequirements, name string, pod *corev1.Pod) {
+	if pod.Annotations == nil {
+		pod.Annotations = map[string]string{}
+	}
+	key := fmt.Sprintf("%s-%s", OriginalCPURequestAnnotation, name)
+	strValue, found := pod.Annotations[key]
```

golangci-lint (2.12.2)

[error] ineffectual assignment to strValue

(ineffassign)